x/ecocredit/client/testsuite/tx.go

```diff
@@ -946,9 +945,8 @@ func (s *IntegrationTestSuite) TestTxRetire() {
 				},
 				s.commonTxFlags()...,
 			),
-			expectErr:       true,
-			errInTxResponse: true,
-			expectedErrMsg:  "abcde is not a valid credit batch denom",
+			expectErr:      true,
+			expectedErrMsg: "denomination didn't match the format",
```

Comment:
nit: Can we change the err message to `invalid denom. Valid denom format is : ****` or something similar

Reply:
@aleem1314 could you address that please?